docs-2.0/5.configurations-and-logs/1.configurations/1.configurations.md

```diff
@@ -61,7 +61,9 @@ curl 127.0.0.1:19779/flags
 
 ## Configuration files
 
-Nebula Graph provides two initial configuration files for each service, `<service_name>.conf.default` and `<service_name>.conf.production`. Users can use them in different scenarios conveniently. The default path is `/usr/local/nebula/etc/`.
+### Configuration files for clusters installed from source, with an RPM/DEB package, or a TAR package
```

Comment:
what does "installed from souce" mean? the source code?

Reply:
yup, means the source code